### ISSUE TITLE
Unable to reproduce the results from the paper

### ISSUE THREAD

**Issue by sina-nabavi** [sunnweiwei/RankGPT#23]
I am unable to reproduce the scores on paper Table 13 using your pre-trained "deberta-10k-rank_net" model. I use the `deberta-10k-rank_net` function, but the scores that I get are lower. 

Moreover, what is the difference between "deberta-v3-large 5K" and "deberta-v3-large 5K label" in the table?
![image](https://github.com/sunnweiwei/RankGPT/assets/30366004/72262421-2419-4b37-b2ac-3dd8bfbfc932)

**Comment by sunnweiwei** [comment]
Hi,

Could you please share the hyperparams used to train the model?

Regarding the figure, the DeBERTa Label 5K (in blue) denotes the model trained using the same hyperparams as the red one but using the MS Marco labels instead of GPT-3.5 predicted labels.
